src/wrapped_radix_engine_toolkit/high_level/ret_api.rs

```diff
@@ -145,29 +145,27 @@ pub fn debug_print_compiled_notarized_intent(
     format!("{:?}", notarized)
 }
 
-/// REQUIRES NETWORK CALL (and probable cache)
-/// Requires kotlinx to be setup
-// #[uniffi::export]
-// pub async fn manifest_assets_transfers(
-//     _transfers: AssetsTransfersTransactionPrototype,
-//     _message: Message,
-// ) -> Result<Manifest> {
-//     unreachable!()
-// }
-
-/// REQUIRES NETWORK CALL (and probable cache)
-/// Requires kotlinx to be setup
-// #[uniffi::export]
-// pub async fn needs_signature_for_depositing(
-//     _into_account: Account,
-//     _resource: ResourceAddress,
-// ) -> Result<bool> {
-//     unreachable!()
-// }
+/// Uses `per_asset_transfers` after having transposed the `PerRecipientAssetTransfers`
+/// into `PerAssetTransfers`. We always use `PerAssetTransfers` when building the manifest
+/// since it is more efficient (allows a single withdraw per resource) => fewer instruction =>
+/// cheaper TX fee for user.
+#[uniffi::export]
+pub fn manifest_per_recipient_transfers(
+    transfers: PerRecipientAssetTransfers,
+) -> TransactionManifest {
+    TransactionManifest::per_recipient_transfers(transfers)
+}
+
+#[uniffi::export]
+pub fn manifest_per_asset_transfers(
+    transfers: PerAssetTransfers,
```

Comment:
wallets need to specify which resources for which recipients require `deposit` vs `try_deposit_or_abort`, part of `PerAssetTransfers `